### ISSUE TITLE
User Login Form Shows Error Container on Load

### ISSUE THREAD

**Issue by twd3** [statamic/cms#1493]
Using the code in the docs for a user login form shows the "if error" container on page load. To reproduce copy/paste code from docs and load page. You should see the bg-red-300 that is part of:
```
    {{ if errors }}
        <div class="bg-red-300 text-white p-2">
            {{ errors }}
                {{ value }}<br>
            {{ /errors }}
        </div>
    {{ /if }}
```

**Comment by jackmcdade** [comment]
What version are you running?

**Comment by twd3** [comment]
Beta 19

**Comment by twd3** [comment]
I also just realized that the ```{{ error:email }}``` is not working for me either...

**Comment by twd3** [comment]
Shoot, this might be similar to my issue: https://github.com/statamic/cms/issues/1080

**Comment by jackmcdade** [comment]
Confirmed parser bug.

**Comment by jesseleite** [comment]
Fixed in 4d4e56c1a2f543bc180b054d629bc98e723ddb1b 👍

**Comment by jesseleite** [comment]
>I also just realized that the {{ error:email }} or {{ error:password }} is not working for me either on this form.

This is also fixed in 3bed8305eb22407469b370d8bcff04d758da84a5.
